www/mikd08/cv6/dbinfo.php

```diff
@@ -0,0 +1,8 @@
+<?php 
+
+const HOST = "localhost";
+const USERNAME = "root";
+const PASSWORD = "";

```

Comment:
na serveru eso zmente konfiguracni udaje tak aby to tam fungovalo 

![image](https://user-images.githubusercontent.com/20724910/228268339-08a3f097-28db-4d75-8c62-c50ac1aa515f.png)